sdks/python/apache_beam/coders/coders.py

```diff
@@ -787,6 +788,40 @@ def as_deterministic_coder(self, step_label, error_message=None):
     return self
 
 
+AVRO_CODER_URN = "beam:coder:avro:v1"
+
+
+class AvroCoder(FastCoder):
+  """A coder used for AvroRecord values."""
+
+  def __init__(self, schema):
+    self.schema = schema
+
+  def _create_impl(self):
+    return coder_impl.AvroCoderImpl(self.schema)
+
+  def is_deterministic(self):
+    # TODO: need to confirm if it's deterministic
```

Comment:
Please resolve this or add a JIRA.

Reply:
created https://issues.apache.org/jira/browse/BEAM-7903